test/unit/UnitTestRouter.ts

```diff
@@ -291,18 +293,67 @@ describe('Router Unit Tests', () => {
         assert.equal(await router.executionWindow(txBlock), TxStatus.ExecutionWindow);
       });
 
-      // it('returns Executed when executes transaction', async () => {
-      //   assert.equal(await router.executionWindow(txBlock), TxStatus.Executed);
-      // });
+      it('returns Executed when executes transaction', async () => {
+        const tx = mockTxStatus(txBlock, TxStatus.Executed);
+        assert.equal(await router.executionWindow(tx), TxStatus.Executed);
+      });
     });
   });
 
-  // Added only basic test for this, not sure how it works exactly
   describe('route()', () => {
-    it('route()', async () => {
-      assert.equal(await router.route(txBlock), TxStatus.BeforeClaimWindow);
-      assert.equal(await router.route(txTimestamp), TxStatus.BeforeClaimWindow);
+    describe(TIMESTAMP_TX, () => {
+      it('returns BeforeClaimWindow status', async () => {
+        const tx = mockTxStatus(txTimestamp, TxStatus.BeforeClaimWindow);
+        assert.equal(await router.route(tx), TxStatus.BeforeClaimWindow);
+      });
+
+      it('returns ClaimWindow status', async () => {
+        const tx = mockTxStatus(txTimestamp, TxStatus.ClaimWindow);
+        assert.equal(await router.route(tx), TxStatus.ClaimWindow);
+      });
+
+      // THESE TESTS DO NOT PASS FOR SOME REASON
```

Comment:
Can you use `xit` instead of commenting tests? It will skip them for now and in future lets fix them.

Reply:
Good suggestion. Done.